cluster/manifests/01-admission-control/config.yaml

```diff
@@ -82,6 +82,15 @@ data:
 {{- else }}
   pod.automatic-topology-spread.pools.dedicated.{{$group}}: "{{ $enabled }}"
 {{- end }}
+{{- end}}
+
+  pod.az-block.zone-name: "{{ .Cluster.ConfigItems.blocked_availability_zone }}"
```

Comment:
This could technically work a bit better, but I'd rather drop `worker-default` instead and just get rid of this thing altogether.